src/styles/themer/docs/withTheme.md

```diff
@@ -0,0 +1,33 @@
+The withTheme higher order component is the best way to use Snacks themes in your own React components. It tracks updates to your theme config and re-renders your components as needed. Theme attributes are passed to your components via a `snacksTheme` prop. Using this or the themer directly allows you to share a theme across snacks components and your own components.
```

Comment:
Is there any reason to use `themer` directly?

Reply:
If someone wants to use Snacks along with a non-react library or to change the theme config.